### PR TITLE
Uyuni 2024.03

### PR DESCRIPTION
**Pull request by raulillo82**

### CODE REVIEW

File: pages/stable-version.html [hunk 1]
```diff
@@ -90,26 +90,26 @@ <h1>Stable version 2024.02</h1>
...
-                <li><a href="../doc/2024.02/uyuni_retail_guide.pdf">Retail Guide</a></li>
-                <li><a href="../doc/2024.02/uyuni_common-workflows_guide.pdf">Common Workflows Guide</a></li>
-                <li><a href="../doc/2024.02/uyuni_specialized-guides_guide.pdf">Specialized Guides Guide</a></li>
+                <li><a href="../doc/2024.03/uyuni_quickstart_guide.pdf">Quickstart Guide</a/></li>
+                <li><a href="../doc/2024.03/uyuni_installation-and-upgrade_guide.pdf">Installation and Upgrade guide</a></li>
+                <li><a href="../doc/2024.03/uyuni_client-configuration_guide.pdf">Client Configuration Guide</a></li>
+                <li><a href="../doc/2024.03/uyuni_administration_guide.pdf">Administration Guide</a></li>
+                <li><a href="../doc/2024.03/uyuni_reference_guide.pdf">Reference Guide</a></li>
+                <li><a href="../doc/2024.03/uyuni_retail_guide.pdf">Retail Guide</a></li>
+                <li><a href="../doc/2024.03/uyuni_common-workflows_guide.pdf">Common Workflows Guide</a></li>
```
Comment (juliogonzalez) [inline pages/stable-version.html:111]: Do we still have this book in the doc?

**Comment by juliogonzalez** [comment]
> Docs still missing.

Didn't the doc squad provided the SR already?

**Comment by juliogonzalez** [comment]
> > Docs still missing.
> 
> Didn't the doc squad provided the SR already?

Confirmed, why had the SR merged: https://build.opensuse.org/package/rdiff/systemsmanagement:Uyuni:Master/uyuni-docs_en?linkrev=base&rev=71, so the docs can be added already.

**Comment by raulillo82** [comment]
Continued here: https://github.com/uyuni-project/uyuni-project.github.io/pull/103
I forgot a "git pull" at the very beginning and it was easier to start from scratch.

**Pull request closed without merging**